woocommerce/woocommerce-ios
# [Product Bundles] Remove unused product bundle properties

Part of #8953


## Description

We added some properties in https://github.com/woocommerce/woocommerce-ios/pull/9024 and https://github.com/woocommerce/woocommerce-ios/pull/9025 that represent settings for Product Bundles. For now, we won't be using or displaying most of those settings in the app. (Internal ref: p1677862753194229-slack-C02KUCFCSFP)

To keep things simple and clean in other layers, we can remove the unused properties for now. We're keeping the properties needed to display the list of bundled items, their titles, their stock statuses, and the correct stock status/quantity for the product bundle (parent product).

Note: The diff shows a new file added at `Networking/Networking/Model/Product/ProductBundleItem.swift`. This is the `ProductBundleItem` model moved from `Networking/Networking/Model/Product/ProductBundle/ProductBundleItem.swift`, with the unused properties removed.

## Testing instructions

Confirm tests pass and you can load the Products list in the app as expected. (These properties are not used in the app yet.)


---
